dev/com.ibm.ws.security.authorization.jacc/original.bnd

```diff
@@ -1,17 +1,29 @@
 #*******************************************************************************
-# Copyright (c) 2022 IBM Corporation and others.
+# Copyright (c) 2022, 2024 IBM Corporation and others.
 # All rights reserved. This program and the accompanying materials
 # are made available under the terms of the Eclipse Public License 2.0
 # which accompanies this distribution, and is available at
 # http://www.eclipse.org/legal/epl-2.0/
-# 
+#
 # SPDX-License-Identifier: EPL-2.0
 #
 # Contributors:
 #     IBM Corporation - initial API and implementation
 #*******************************************************************************
 
-Bundle-Name: WAS Security JACC 1.5 Service
-Bundle-SymbolicName: com.ibm.ws.security.authorization.jacc
+Bundle-Name: WAS Security JACC Service
+Bundle-SymbolicName: com.ibm.ws.security.authorization.jacc.common
```

Comment:
That is correct.  The package name still starts the bundle name so it can still be found whereas the transformed.bnd, 1.5.bnd and 2.0.bnd bundle names do not start with the project name which is why they are included in the cnf/build.bnd and in the testpath for the ejb and web bundle to point to the project that those bundles are from.